graph/types/media.graphqls

```diff
@@ -0,0 +1,17 @@
+# Maybe make a basePost interface and extend
+# Media can also just be data on a post
+type Media {
+    id: ID!
+    createdAt: String!
+    isDeleted: Boolean!
+    deletedReasonCode: PostDeletedReason
+    mediaType: String!
+    mediaSize: MediaSize!
+    assetLocation: String!
+}
+
+type MediaSize {
+    height: Int!
+    width: Int!
+    storage: Int! # Should this be in bytes, kbs, something else?
```

Comment:
Hmm, good question. Maybe in kb? I'd recommend changing the name of the field to something other than `storage` as that could be misinterpreted. Perhaps size?